rest_api/operator_apis/ingresscontroller-operator-openshift-io-v1.adoc

```diff
@@ -281,21 +281,21 @@ Required::
 
 | `type`
 | `string`
-| type is the publishing strategy to use. Valid values are: 
- * LoadBalancerService 
- Publishes the ingress controller using a Kubernetes LoadBalancer Service. 
- In this configuration, the ingress controller deployment uses container networking. A LoadBalancer Service is created to publish the deployment. 
- See: https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer 
- If domain is set, a wildcard DNS record will be managed to point at the LoadBalancer Service's external name. DNS records are managed only in DNS zones defined by dns.config.openshift.io/cluster .spec.publicZone and .spec.privateZone. 
- Wildcard DNS management is currently supported only on the AWS, Azure, and GCP platforms. 
- * HostNetwork 
- Publishes the ingress controller on node ports where the ingress controller is deployed. 
- In this configuration, the ingress controller deployment uses host networking, bound to node ports 80 and 443. The user is responsible for configuring an external load balancer to publish the ingress controller via the node ports. 
- * Private 
- Does not publish the ingress controller. 
- In this configuration, the ingress controller deployment uses container networking, and is not explicitly published. The user must manually publish the ingress controller. 
- * NodePortService 
- Publishes the ingress controller using a Kubernetes NodePort Service. 
+| type is the publishing strategy to use. Valid values are:
+ * LoadBalancerService
+ Publishes the ingress controller using a Kubernetes LoadBalancer Service.
+ In this configuration, the ingress controller deployment uses container networking. A LoadBalancer Service is created to publish the deployment.
+ See: https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer
+ If domain is set, a wildcard DNS record will be managed to point at the LoadBalancer Service's external name. DNS records are managed only in DNS zones defined by dns.config.openshift.io/cluster .spec.publicZone and .spec.privateZone.
+ Wildcard DNS management is currently supported only on the AWS, Azure, and GCP platforms.
+ * HostNetwork
+ Publishes the ingress controller on node ports where the ingress controller is deployed.
+ In this configuration, the ingress controller deployment uses host networking, bound to node ports 80 and 443. The user is responsible for configuring an external load balancer to publish the ingress controller via the node ports.
```

Comment:
This becomes partly true after we implement this change. I'm not sure if it's worth hightlighting though. The paragraph is already pretty dense

Reply:
@sherine-k this content already exists in the docs , do you want me to remove this paragraph?

Reply:
No, definitely don't remove it. 
`...bound to node ports 80 and 443`: it could be interesting to specify something like : `..bound to node ports 80 and 443 by default, or port numbers set in .spec.endpointPublishingStrategy.hostNetwork `